python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -551,6 +609,7 @@ class NearestNeighbors(Base,
             X = self.X_m
             n_neighbors += 1
 
+        # TODO: Important that this value does not change for RBC
```

Comment:
what value is this referring to? Will this be solved in the PR or should we open an issue?

Reply:
Ah yes, this was a little note to myself that I need to remove the `k` argument from the index building functions.